packages/components/simple-table/src/simple-table.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { filterDataAttributes } from '@commercetools-uikit/utils';
+import { TableGrid, Header, Body, Row } from './simple-table.styles';
+import { HeaderCell, DataCell } from './cell';
+
+const SimpleTable = props => (
+  <TableGrid
+    {...filterDataAttributes(props)}
+    columns={props.columns}
+    maxHeight={props.maxHeight}
+    maxWidth={props.maxWidth}
+  >
+    <Header isSticky={props.isHeaderSticky}>
+      <Row>
+        {props.columns.map((column, columnIndex) => (
+          <HeaderCell
+            key={`${columnIndex}-${column.key}`}
+            role="column-header"
+            isCondensed={props.isCondensed}
+            /* Sorting Props */
+            sortedBy={props.sortedBy}
+            columnKey={column.key}
+            isSortable={column.isSortable}
+            onSortChange={props.onSortChange}
+            sortDirection={props.sortDirection}
+          >
+            {column.label}
+          </HeaderCell>
+        ))}
+      </Row>
+    </Header>
+    <Body>
+      {props.rows.map((row, rowIndex) => (
+        <Row
+          key={row.id}
+          onClick={
+            props.onRowClick ? () => props.onRowClick(row, rowIndex) : undefined
+          }
+        >
+          {props.columns.map(column => (
+            <DataCell
+              key={`${rowIndex}-{row.id}/${column.key}`}
```

Comment:
Why the rowIndex if we have a row.id? Isn't the row.id enough? BTW: who generates this id?

Reply:
No particular reason actually...

The ID is given by the consumer (and the only required field of the `rows` objects). Usually it's already part of the items that you want to render on the table and not for displaying.

Anyway, I'll remove the `rowIndex`